docs/customization/menu.rst

```diff
@@ -238,23 +245,32 @@ type is default to make the example easily customizable.
 
```

Comment:
Before this example and the above one, we should describe what exactly are we doing.

```
In the example below, we are adding one new button to the Admin Order Show Menu. It is a ``transition`` type button, that will let the admin to fulfill the order.
```